.buildkite/scripts/run-installer-compliance.sh

```diff
@@ -4,14 +4,16 @@ set -euo pipefail
 
 WORKSPACE="$(pwd)"
 
-source .buildkite/scripts/install_deps.sh
-add_bin_path
+if [[ "${CI:-"false"}" == "true" ]]; then
+    source .buildkite/scripts/install_deps.sh
+    add_bin_path
 
-echo "--- Install Go :go:"
-with_go
+    echo "--- Install Go :go:"
+    with_go
 
-echo "--- Install docker-compose"
-with_docker_compose
+    echo "--- Install docker-compose"
+    with_docker_compose
+fi
```

Comment:
Adding this, it can be used this same script locally to test 
```shell
# some tests require to run from a fresh stack
elastic-package stack down -v 
.buildkite/scripts/run-installer-compliance.sh 8.15.0-SNAPSHOT 3.2.0
```